aspnetcore/security/authentication/accconfirm.md

```diff
@@ -1,38 +1,66 @@
 ---
-title: Account Confirmation and Password Recovery
+title: Account Confirmation and Password Recovery in ASP.NET Core
 author: rick-anderson
 description: Shows how to build an ASP.NET Core app with email confirmation and password reset.
 keywords: ASP.NET Core, password reset, email confirmation, security
 ms.author: riande
 manager: wpickett
-ms.date: 06/14/2017
+ms.date: 07/19/2017
 ms.topic: article
-ms.assetid: d794500b-86f7-4229-a237-e0dd00e2dc08
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: security/authentication/accconfirm
 ---
-# Account confirmation and password recovery
-
-<a name=security-authentication-account-confirmation></a>
+# Account confirmation and password recovery in ASP.NET Core
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 This tutorial shows you how to build an ASP.NET Core app with email confirmation and password reset.
 
 ## Create a New ASP.NET Core Project
 
-The tutorial requires Visual Studio 2017 or higher.
+# [ASP.NET Core 2.x](#tab/aspnet20)
+
+The tutorial requires Visual Studio 2017 preview 2 or later.
+
+* In Visual Studio, create a New Web Application Project.
+* Select **ASP.NET Core 2.0**.
+* Select **Change Authentication** and set to **Individual User Accounts**.
+* Keep the default **Store user accounts in-app**.
+
+![New Project dialog showing "Individual User Accounts radio" selected](accconfirm/_static/2.png)
+
+# [ASP.NET Core 1.x](#tab/aspnet1x)
+
+The tutorial requires Visual Studio 2017 or later.
 
 * In Visual Studio, create a New Web Application Project.
 * Select **Change Authentication** and set to **Individual User Accounts**.
 
 ![New Project dialog showing "Individual User Accounts radio" selected](accconfirm/_static/indiv.png)
 
+---
+
+### CLI project creation for macOS and Linux
+
+If you're using the CLI or SQLite, run the following in a command window:
+
+```
+dotnet new mvc --auth Individual
+```
+
+* `--auth Individual`: specifies the Individual User Accounts template.
+* On Windows, add the `-uld` option. The `-uld` option creates a LocalDB connection string rather than a SQLite DB.
+* run `new mvc --help` to get help on this command.
+
+## Test new user registration
+
 Run the app, select the **Register** link, and register a user. Follow the instructions to run Entity Framework migrations. At this  point, the only validation on the email is with the [[EmailAddress]](http://msdn.microsoft.com/library/system.componentmodel.dataannotations.emailaddressattribute(v=vs.110).aspx) attribute. After you submit the registration, you are logged into the app. Later in the tutorial we'll change this so new users cannot log in until their email has been validated.
```

Comment:
Should this say "Entity Framework Core migrations" to clarify that EF 6 isn't being used?